punish/punish.py

```diff
@@ -594,6 +611,45 @@ def check(m):
         if ctx.invoked_subcommand is None:
             await self.bot.send_cmd_help(ctx)
 
+    @punishset.command(pass_context=True, name="remove_role_list")
+    async def punishset_remove_role_list(self, ctx, *, rolelist=None):
+        """Set what roles to remove when punishing.
+        COMMA SEPARATED LIST (e.g. Admin,Staff,Mod)
+
+        To get current remove role list, run command with no roles.
+
+        Add role_list_clear as the role to clear the server's remove role list.
+        """
+        server = ctx.message.server
+        if rolelist is None:
+            roles = self.json.get(server.id, {}).get("REMOVE_ROLE_LIST")
+            if roles:
+                await self.bot.say("List of roles to remove when punishing: {}".format(roles))
+            else:
+                await self.bot.say("No roles defined for removal.")
+            return
+        elif "role_list_clear" in rolelist:
+            await self.bot.say("Remove role list cleared.")
+            self.json[server.id]["REMOVE_ROLE_LIST"] = []
+            self.save()
+            return
+
+        unparsed_roles = list(map(lambda r: r.strip(), rolelist.split(',')))
+        parsed_roles = list(map(lambda r: self._role_from_string(server, r),
+                                unparsed_roles))
+
+        if None in parsed_roles:
```

Comment:
After parsing, also ensure that the everyone role isn't in the list. A call to remove it will always fail.